introspection-engine/introspection-engine-tests/tests/tables/mod.rs

```diff
@@ -0,0 +1,697 @@
+
+    Ok(())
+}
+
+// SQLite does not have a serial type that's not a primary key.
+#[test_each_connector(ignore("sqlite"))]
+async fn a_table_with_non_id_autoincrement(api: &TestApi) -> crate::TestResult {
+    api.barrel()
+        .execute_with_schema(
+            |migration| {
+                migration.create_table("Test", |t| {
+                    t.add_column("id", types::integer().primary(true));
+                    t.add_column("authorId", types::serial().unique(true));
+                });
+            },
+            api.schema_name(),
+        )
+        .await?;
+
+    let dm = indoc! {r#"
+        model Test {
+            id       Int @id
+            authorId Int @default(autoincrement()) @unique
+        }
+    "#};
+
+    assert_eq_datamodels!(dm, &api.introspect().await?);
+
+    Ok(())
+}
+
+#[test_each_connector]
+async fn default_values(api: &TestApi) -> crate::TestResult {
+    api.barrel()
+        .execute_with_schema(
+            |migration| {
+                migration.create_table("Test", |t| {
+                    t.add_column("id", types::primary());
+                    t.add_column(
+                        "string_static_char",
+                        types::custom("char(5)").default("test").nullable(true),
+                    );
+                    t.add_column(
+                        "string_static_char_null",
+                        types::r#char(5).default(types::null()).nullable(true),
+                    );
+                    t.add_column(
+                        "string_static_varchar",
+                        types::varchar(5).default("test").nullable(true),
+                    );
+                    t.add_column("int_static", types::integer().default(2).nullable(true));
+                    t.add_column("float_static", types::float().default(1.43).nullable(true));
+                    t.add_column("boolean_static", types::boolean().default(true).nullable(true));
+                    t.add_column(
+                        "datetime_now",
+                        types::datetime().default(functions::current_timestamp()).nullable(true),
+                    );
+                });
+            },
+            api.schema_name(),
+        )
+        .await?;
+
+    let dm = indoc! { r#"
+        model Test {
+            id                      Int       @id @default(autoincrement())
+            string_static_char      String?   @default("test")
+            string_static_char_null String?
+            string_static_varchar   String?   @default("test")
+            int_static              Int?      @default(2)
+            float_static            Float?    @default(1.43)
+            boolean_static          Boolean?  @default(true)
+            datetime_now            DateTime? @default(now())
+        }
+    "#};
+
+    assert_eq_datamodels!(dm, &api.introspect().await?);
+
+    Ok(())
+}
+
+#[test_each_connector(capabilities("enums"))]
```

Comment:
move this to enums pls

Reply:
This was actually a duplicate test and `enums/mod.rs` already has one that tests the same thing.